docs/products/kafka/kafka-connect/getting-started.rst

```diff
@@ -0,0 +1,41 @@
+Getting started
+===============
+
+Aiven services are managed in the Aiven `web console <https://console.aiven.io/>`__ . When you first log in to the console with your email address and password, you will see the **Services** view, which shows you all the services in the currently selected project.
+
+.. Warning::
+
+    Aiven for Apache Kafka Connect services can be created only on top of existing Aiven for Apache Kafka running services. 
+    If your project doesn't contain any Aiven for Apache Kafka services, create one before attempting to create an Aiven for Apache Kafka Connect service.
+
+Create a new Aiven for Apache Kafka Connect dedicated service:
```

Comment:
I wonder if adding a title and bringing this into a subsection will help to scan content quickly

Creating Aiven for Apache Kafka Connect
------------------------------------------

Follow these steps to create a new Aiven for Apache Kafka Connect dedicated service:

Reply:
Or is it too much extra info?

Reply:
I added as dedicated service